packages/client-python/src/rocketride/_engine/manager.py

```
@@ -0,0 +1,251 @@
+
+
+logger = logging.getLogger('rocketride')
+
+
+class EngineManager:
+    """High-level engine lifecycle manager.
+
+    Used by RocketRideClient for auto-spawn and by the CLI ``run`` command.
+    """
+
+    def __init__(self):
```

Ruff (0.15.7)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)